trento/migration/systemd-install.md

```diff
@@ -487,12 +435,78 @@ This is a basic guide for creating a self-signed certificate for use with Trento
 
    If there are issues with the configuration, the output will indicate what needs to be adjusted.
 
-1. Reload Nginx to apply changes:
+1. If SSL certificates were modified or added, reload NGINX to apply changes.
 
    ```bash
    systemctl reload nginx
    ```
 
+## Prepare SSL certificate for NGINX
+
+Create or provide a certificate for [NGINX](https://nginx.org/en/) to enable SSL for Trento.
+This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the [OpenSSL documentation](https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html).
```

Comment:
Hello @dmpop . Where does exactly "to the very end" mean? The very end of what section?
Also "Instead of creating a self-signed certificate, you use": a "can" or "may" is missing ,night?